Options.lua

```diff
@@ -5431,10 +5434,10 @@ do
                                     args = {
                                         damagePets = {
                                             type = "toggle",
-                                            name = "Enemies Damaged by Minions",
+                                            name = "Include Enemies Damaged By Your Pets and Minions",
```

Comment:
`Include Enemies Damaged by Your Pets and Minions`

(See https://titlecaseconverter.com/.)

Reply:
english hard